.circleci/continue_config.yml

```diff
@@ -42,6 +43,14 @@ executors:
   check:
     docker:
       - image: prestocpp/velox-check:mikesh-20210609
+  macos-intel:
+    macos:
+      xcode: "14.3.0"
+    resource_class: macos.x86.medium.gen2
+  macos-m1:
```

Comment:
it looks like we are not using m1 so we'd probably better remove macos-m1 here

Reply:
Ok.